### DISCUSSION TITLE
failed to fetch

### DISCUSSION THREAD

**Discussion by starfort** [AntlerVC/firetable#1083] [Support / Q&A] [answered]
I keep getting failed to fetch while trying to add an existing firebase project to my first workspace.

**Comment by BeeBombshell** [comment]
Hey @starfort, could you share some screenshots or a video recording of the issue?

**Reply by starfort** [reply]
Sure
<img width="1440" alt="Screen Shot 2023-01-19 at 1 35 51 PM" src="https://user-images.githubusercontent.com/122779462/213567739-9eed44a6-ab17-4506-add7-5ab86e7709bf.png">

**Reply by starfort** [reply]
and then it doesn't find any projects despite having one

**Comment by BeeBombshell** [comment]
Hey @starfort, I discussed this issue with the team and it's apparently an issue on the backend. We're investigating this issue, and we'll get back to you with a solution soon. Thanks for bringing this up!

**Reply by BeeBombshell** [reply]
Hey @starfort, this issue has now been fixed from our end. Hope it has been resolved for you now. Closing this one.

**Comment by sigmet** [comment]
I have the same issue: I cannot connect Rowy to existing projects nor can I create new ones within Rowy.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @sigmet, as a workaround for this issue, you can follow the steps below:

STEP 1: Go to this page: https://myaccount.google.com/permissions?continue=https%3A%2F%2Fmyaccount.google.com%2Fsecurity and remove Rowy access.
STEP 2: Logout in the setup section, and re-authenticate.

**Comment by ayteksokmen** [comment]
I have the same issue. I can see my existing projects and rowy can create too. But when I select and tap confirm, it gives me the same error: Failed to fetch. Also, removing access and re-authenticate didn't help.

**Reply by bofeiw** [reply]
We have pushed a fix to this. Please let us know if you still have issues setting up projects.
